[FIX] Removed PyPDF2 library from external dependencies

luiseevaquer opened the pull request:
In the requirement file of odoo/odoo the library is pinned to version 1.26.0
    (https://github.com/odoo/odoo/commit/021de21e8cdcf35276c945d8070d04435fa1a19c),
    if library is unpinned, it is forced to install the current version, which
    generates the following error when running odoo:

    AttributeError: module 'PyPDF2' has no attribute 'filters'

luisg123v reviewed (changes requested):
Since requirements are built up from manifest's requirements, you need to remove dependencies also from there.

In addition, the `requests` requirement is in the same situation.

CC @suniagajose

suniagajose commented:
> Since requirements are built up from manifest's requirements, you need to remove dependencies also from there.
> 
> In addition, the `requests` requirement is in the same situation.
> 
> CC @suniagajose

Complementing your comment @luisg123v 

https://github.com/odoo/odoo/blob/14.0/requirements.txt#L38

etobella commented:
requirements.txt is generated automatically from manifests. You need to remove the library from external_dependancies key on the manifest

etobella commented:
Or pin it there to the expected value

luisg123v reviewed (changes requested):
Hi @lescobarvx,

- You need to perform a commit per module. That's because, when modules are migrated, history is brought taking commits related to that module. So, if you leave only only commit, if both modules are migrated, commit will be duplicated in next version
- Missing removing `requests`, that is in the module `account_invoice_download_ovh`

Commit:
[FIX] requirements.txt: Removed PyPDF2 library

    In the requirement file of odoo/odoo the library is pinned to version 1.26.0
    (https://github.com/odoo/odoo/commit/021de21e8cdcf35276c945d8070d04435fa1a19c),
    if library is unpinned, it is forced to install the current version, which
    generates the following error when running odoo:

    AttributeError: module 'PyPDF2' has no attribute 'filters'

Commit:
[FIX] pdf_helper: Removed PyPDF2 library from external dependencies

    This library is removed because it is already pinned in the
    requirements.txt file of odoo/odoo

    https://github.com/odoo/odoo/blob/14.0/requirements.txt#L38

Commit:
[FIX] base_ubl: Removed PyPDF2 library from external dependencies

    This library is removed because it is already pinned in the
    requirements.txt file of odoo/odoo

    https://github.com/odoo/odoo/blob/14.0/requirements.txt#L38

Commit:
[FIX] account_invoice_download_ovh: Removed requests library from external dependencies

    This library is removed because it is already pinned in the
    requirements.txt file of odoo/odoo

    https://github.com/odoo/odoo/blob/14.0/requirements.txt#L46

luisg123v reviewed (approved):
LGTM :+1: 

@moylop260 could you review, please?

moylop260 reviewed (approved):
👍

moylop260 commented:
The CI is even green and the PR is already approved so...

moylop260 commented:
/ocabot merge minor
